src/app/zap-templates/zcl/data-model/silabs/general.xml

```diff
@@ -205,6 +205,9 @@ limitations under the License.
     <command source="server" code="0x02" name="Leave" optional="false">
       <description>The Leave event SHOULD be emitted by a Node prior to permanently leaving the Fabric.</description>
     </command>
+    <command source="client" code="0xa0" name="MfgSpecificPing" optional="true">
+      <description>MfgSpecificPing without manufacturer namespace before we fill the gap for MfgCode</description>
+    </command>
```

Comment:
That sounds wrong. We are basically converting a manufacturer specific command as a spec'ed command. Then folks will complain that the Basic cluster does not follow the spec...

Reply:
#6486 tracks for MEI prefix, will not include this change.

Reply:
Can you add a specific  issue number next to this change ? `<!-- Issue #???? Support MfgSpecificPing (manufacturer specific command) using IM -->` ?
Otherwise I'm pretty sure someone will arrive in a few days and remove this stuff just because this is not part of the spec. And obviously manufacturer specific commands are not spec'ed.

Reply:
Done